services/trainer/README.md

```diff
@@ -0,0 +1,15 @@
+# Trainer Service
+
+This is the directory for the Trainer service, which takes some model training
+metadata, and trains/uploads the resulting model to cloud storage for future
+transcription jobs.
+
+## Installation
+
+- Make sure you have poetry installed, and a compatible python version.
+- Create a virtual env and install the dependencies: `poetry install`
+
+## Running the tests
+
+- Unit tests: `poetry run pytest -m 'not integration'`
```

Comment:
`'not integration'`? 😂

Reply:
Runs all the tests except those marked as integration tests :) 
I guess I could also mark all the unit tests as unit but that seems boilerplatey